src/gen.ts

```diff
@@ -38,6 +29,30 @@ export function genDynamicImport (specifier: string, opts: DynamicImportOptions
 // genImportName
 
 // Internal
+
+function genStatement (type: 'import' | 'export', specifier: string, names?: Name | Name[], opts: CodegenOptions = {}) {
+  const specifierStr = genString(specifier, opts)
+  if (!names) {
+    return `${type} ${specifierStr};`
+  }
+
+  const nameArray = Array.isArray(names)
+
+  const _names = (nameArray ? names : [names]).map((i: Name) => {
+    if (typeof i === 'string') { return { name: i } }
+    if (i.name === i.as) { i = { name: i.name } }
+    // TODO: Ensure `name` and `as` are valid identifiers
+    // TODO: Ensure `as` is provided for default import
```

Comment:
Note: It might lead to a hacky situation in the future to generate default import/exports `export default from` vs `import X from`